ramalama/cli.py

```diff
@@ -190,6 +190,11 @@ def post_parse_setup(args):
     if hasattr(args, "runtime_args"):
         args.runtime_args = shlex.split(args.runtime_args)
 
+    if hasattr(args, "threads") and args.threads < 0:
+        nproc = os.cpu_count()
```

Comment:
Can you default this in the CLI not at this point.  Basically if the user does not specify the --threads command it should show a default of nproc/2

Reply:
Also need to update the man page.

Reply:
The thing about defaulting it directly in the CLI, is it makes the documentation and configurations kinda funny.

You are trying to kinda say: "if you want to use an automatically assigned number use -1, that's the true default although we don't display it, I know the CLI says the default is 6 but that's only kinda true and it depends on the machine."

Reply:
Changed